hw/ip/flash_ctrl/data/flash_ctrl_testplan.hjson

```diff
@@ -24,199 +24,356 @@
       tests: ["flash_ctrl_smoke"]
     }
     {
-      name: backpressure
+      name: smoke_hw_rd
       desc: '''
-            Sanity + randomly pick between back-pressured reads / writes and fifo status-polled
-            reads / writes via the controller.
+            Perform host direct read on the single page of one bank within Data partition.
+            '''
+      milestone: V1
+      tests: []
+    }
+    {
+      name: sw_op_pwrmgr
+      desc: '''
+            Perform flash protocol controller read, program and erase on the single page of one
+            bank within Data partition. Finally perform read on same location in order to test
+            if previous operation was done successfully. Observe pwrmgr interface in order to
```

Comment:
I think the idle output for pwrmgr should be tested always, not just in a specific test, which is why I suggest an SVA or some other check that is always running (like in the base_vseq).

Reply:
Agree on that.

Reply:
removed from testplan since it will be SVA most probably.